### PR TITLE
test fixing

### PR DESCRIPTION
**Pull request by chris1984**

### CODE REVIEW

File: test/functional/content_view/version/republish_repositories_test.rb [hunk 1]
```diff
@@ -26,7 +26,7 @@ module HammerCLIKatello
       expect_foreman_task('3')
 
       result = run_cmd(%w(content-view version republish-repositories --organization org1
-                          --content-view cv --version 1))
+                          --content-view cv --version 1 --force true))
```
Comment (ofedoren) [inline test/functional/content_view/version/republish_repositories_test.rb:29]: Isn't it dangerous that few checks wont apply? I don't know what this flag is for exactly, but doesn't it mean that if this flag is set then some logic in the command is dropped?

File: test/functional/package/list_test.rb [hunk 2]
```diff
@@ -72,23 +72,15 @@ module HammerCLIKatello
     describe 'environment options' do
       it 'may be specified environment name' do
         org_id = 2
-        env_name = "dev"
+        env_id = 3
         expected_id = 6
-        expect_lifecycle_environment_search(org_id, env_name, expected_id)
+        expect_lifecycle_environment_search(org_id, env_id, expected_id)
```
Comment (ofedoren) [inline test/functional/package/list_test.rb:77]: If it searches for environment why you drop the name here? I assume that it does `index` call with the name to grab the id.

File: test/functional/capsule/content/remove_lifecycle_environment_test.rb [hunk 3]
```diff
@@ -15,14 +16,14 @@ class Content
...
-          run_cmd(%w(capsule content remove-lifecycle-environment --id 1 --environment-id 2))
+          run_cmd(%w(capsule content remove-lifecycle-environment --id 1 --lifecycle-environment-id 2))
         end
 
         it 'resolves environment ID from name' do
           expect_lifecycle_environment_search('3', 'env2', 2)
           expect_lifecycle_environment_remove('id' => 1, 'environment_id' => 2)
 
-          run_cmd(%w(capsule content remove-lifecycle-environment --id 1 --environment env2
+          run_cmd(%w(capsule content remove-lifecycle-environment --id 1 --lifecycle-environment env2
```
Comment (ofedoren) [inline test/functional/capsule/content/remove_lifecycle_environment_test.rb:26]: If you're dropping support for `--environment` in tests, maybe it's time to actually drop it from the commands?

Comment (chris1984) [reply test/functional/capsule/content/remove_lifecycle_environment_test.rb:26]: @ofedoren I was just trying to fix the tests, do you think we should keep it?

File: test/functional/package/list_test.rb (hunk 2 above)
Comment (chris1984) [reply test/functional/package/list_test.rb:77]: ok fair point I will switch it back

File: test/functional/content_view/version/republish_repositories_test.rb (hunk 1 above)
Comment (chris1984) [reply test/functional/content_view/version/republish_repositories_test.rb:29]: Sadly, this command now won't work unless --force is passed in. When I try it without it, it companies it needs and when i set it to false, it says it must be true. This is the pr that introduced it.

https://github.com/Katello/katello/commit/79e606ac3ffcb416c3c03baeca12e083e24bbe84

Comment (ofedoren) [reply test/functional/content_view/version/republish_repositories_test.rb:29]: Ah, right. Well, if it *is* required in the API in such way, then it's OK, my bad :/

File: test/functional/module_stream/info_test.rb [hunk 4]
```diff
@@ -11,11 +11,11 @@ module HammerCLIKatello
 
     it 'resolves ID from name and repo id' do
       api_expects(:module_streams, :index)
-        .with_params(search: "name = \"duck\"", name: "duck", repository_id: "1")
-        .returns(index_response([{'id' => 1}]))
+        .with_params(search: "name = \"duck\"", name: "duck", repository_id: 1)
```
Comment (parthaa) [inline test/functional/module_stream/info_test.rb:14]:
```suggestion
        .with_params(search: "name = \"duck\"", name: "duck", repository_id: "1")
```

**Pull request closed without merging**